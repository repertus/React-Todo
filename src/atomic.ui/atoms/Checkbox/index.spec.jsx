import React from 'react';
import renderer from 'react-test-renderer';

import CheckBox from './index';

describe('CheckBox', () => {
  test('Toggle box', () => {
    const component = renderer.create(<CheckBox />);
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();

    // Toggle off check
    tree.props.onChange({
      target: { checked: false },
    });
    // Re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
