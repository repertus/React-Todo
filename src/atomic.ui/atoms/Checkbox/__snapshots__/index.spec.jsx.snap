// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox Toggle box 1`] = `
<input
  checked={true}
  onChange={[Function]}
  type="checkbox"
/>
`;

exports[`CheckBox Toggle box 2`] = `
<input
  checked={false}
  onChange={[Function]}
  type="checkbox"
/>
`;
